Wrappers.h

```diff
@@ -0,0 +1,38 @@
+#pragma once
+#include "Band.h"
+#include "Clothing.h"
```

Comment:
You aren't using any of your types in this header file, which means they don't need to be included.
The whole point of this project part is "Don't Include what you don't use".

You are including header files for types that aren't being used.  you aren't accessing any of their member variables nor are you calling any of their member functions (this includes constructors and destructors).  This means you aren't **using** them, and they can safely be forward-declared.

This header file should include NOTHING.
Forward declare everything in this header file.
`#Include` in the cpp file.
- [ ] revise